docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1697,7 +1707,8 @@ client can provide a value for a P4~16~ expression. `P4DataTypeSpec` describes
 the compile-time of the expression as a Protobuf `oneof`, which can be:
 
 * a string representing the name of the type in case of a named type (`struct`,
-  `header`, `header_union`, `enum` or `serializable_enum`),
+  `header`, `header_union`, `enum`, `serializable_enum` or user-defined "new"
+  `type`),
```

Comment:
This comment is not for this change, but for original line 1697, new line 1707 above where it says "the compile-time of".  Maybe that should be "the compile-time type of" ?